mediaengine.go

```diff
@@ -209,6 +209,23 @@ func NewRTPVP8Codec(payloadType uint8, clockrate uint32) *RTPCodec {
 	return c
 }
 
+// NewRTPVP8CodecExt is a helper to create an VP8 codec
+func NewRTPVP8CodecExt(payloadType uint8, clockrate uint32) *RTPCodec {
+	c := NewRTPCodecExt(RTPCodecTypeVideo,
+		VP8,
+		clockrate,
+		0,
+		"",
+		payloadType,
+		[]RTCPFeedback{
```

Comment:
Can you have `RTCPFeedback` be passed as a function argument? In case we want to support other congestion control algorithms.

Reply:
OK